### ISSUE TITLE
v0.11.0 release notes has docker tag typos

### ISSUE THREAD

**Issue by cchadowitz-pf** [jolibrain/deepdetect#974]
Not a big issue, but on the Github release page for v0.11.0 the docker tags are missing `v` in front of the version number:

```
Docker images:
CPU version: docker pull jolibrain/deepdetect_cpu:0.11.0
GPU (CUDA only): docker pull jolibrain/deepdetect_gpu:0.11.0
GPU (CUDA and Tensorrt) :docker pull jolibrain/deepdetect_cpu_tensorrt:0.11.0
GPU with torch backend: docker pull jolibrain/deepdetect_gpu_torch:0.11.0
All images available on https://hub.docker.com/u/jolibrain
```

Instead of `...:0.11.0` they should end with `...:v0.11.0`. This can be verified by checking the tags on dockerhub.

**Comment by tlok666** [comment]
Hi, I have installled the jolibrain/deepdetect_gpu_torch:v0.11.0, but the torch only works on cpu

I have two TITAN Xp.

How can I make the gpu torch work?

**Comment by sileht** [comment]
I manually fixed the 0.11 releases.
Future releases will not have the issue anymore.
